### DISCUSSION TITLE
Transformations of different frames in JSBSim

### DISCUSSION THREAD

**Discussion by cxr001** [JSBSim-Team/jsbsim#847] [Q&A]

**Comment by seanmcleod** [comment]
Where is the question or comment?

**Reply by cxr001** [reply]
Emm, I am trying to transform a LLA coordination (geographic) into a local NED frame with a specific origin. I can find a method to transform LLA into ECEF by UE GeoReferencing system, but I don't know how to do the next step.

**Comment by seanmcleod** [comment]
Take a look at - https://en.wikipedia.org/wiki/Local_tangent_plane_coordinates for a formula relating NED and ECEF coordinates.

**Reply by cxr001** [reply]
When I use GNC Utilities, I found that if I set guidance/target_wp_latitude or longitude bigger than 90 degree, it would cause a run time error. Could I remove this restriction?

**Comment by seanmcleod** [comment]
What does it mean for latitude to be greater than 90 degrees?

**Reply by cxr001** [reply]
for example, if I set "guidance/target_wp_latitude_rad" = 2 or -2, by command console, it would cause a run time error. But if I set "guidance/target_wp_latitude_rad" = 1.5, my aircraft will fly to the waypoint normally.

**Comment by seanmcleod** [comment]
What is the maximum and minimum latitude on earth?

**Reply by cxr001** [reply]
emm,sorry, I am wrong here
